src/webui/webapplication.h

```diff
@@ -54,7 +54,7 @@
 #include "base/utils/version.h"
 #include "api/isessionmanager.h"
 
-inline const Utils::Version<3, 2> API_VERSION {2, 11, 3};
+inline const Utils::Version<3, 2> API_VERSION {2, 12, 0};
```

Comment:
It is non-breaking change so 3rd number should be increased in this case.

Reply:
In qBittorrent WebAPI version first number is reserved for deepest redesign indication, the second one is to indicate breaking changes (i.e. the ones that break current clients until they add support for it, e.g. changing or deleting something), and the third one is to indicate non-breaking changes (i.e. the ones that don't break current clients, e.g. adding something).